test/e2e/storage/csi_mock_volume.go

```diff
@@ -684,7 +684,9 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 				sc, pvc, pod := createPod(pvcReference)
 				gomega.Expect(pod).NotTo(gomega.BeNil(), "while creating pod for resizing")
 
-				framework.ExpectEqual(*sc.AllowVolumeExpansion, true, "failed creating sc with allowed expansion")
+				if !*sc.AllowVolumeExpansion {
+					framework.Failf("failed creating sc with allowed expansion")
```

Comment:
Please use `framework.Fail` instead of `framework.Failf` when there is no need for printf-style formatting.

This applies also in several other places.

Reply:
Updated. Please have a look.